src/platform.ts

```diff
@@ -211,17 +209,29 @@ export class HomebridgeLgThinqPlatform implements DynamicPlatformPlugin {
       // something globally unique, but constant, for example, the device serial
       // number or MAC address
       const uuid = this.api.hap.uuid.generate(device.deviceId)
+      const modelInfo = await this.thinqApi!.getDeviceModelInfo(
+        device.modelJsonUri,
+      )
 
       const matchingAccessories = this.accessories.filter(
         (accessory) => accessory.UUID === uuid,
       )
 
       if (matchingAccessories.length > 0) {
         this.log.info('Existing accessory:', device.alias)
+
         // check that the device has not already been registered by checking the
         // cached devices we stored in the `configureAccessory` method above
         for (const accessory of matchingAccessories) {
+          accessory.context = {}
           accessory.context.device = device
+          accessory.context.modelInfo = modelInfo
+
+          new LgAirConditionerPlatformAccessory(
+            this,
+            accessory,
+          ).updateCharacteristics()
```

Comment:
This shouldn't be necessary -- we already call `updateCharacteristics()` in the class constructor.

If this ended up fixing something, `updateCharacteristics()` should likely be moved down in the `LgAirConditionerPlatformAccessory ` constructor instead

```suggestion
          new LgAirConditionerPlatformAccessory(this, accessory)
```